shell/platform/darwin/macos/framework/Source/FLEViewController.mm

```diff
@@ -584,4 +597,16 @@ - (void)scrollWheel:(NSEvent*)event {
   [self dispatchMouseEvent:event phase:kHover];
 }
 
+- (void)onSettingsChanged:(NSNotification*)notification {
```

Comment:
This method needs a declaration comment, per style guide. The macOS shell uses the style of pre-declaring all internal methods and putting the declaration there, so please follow that style. (~line 90 in this file)

Also, the method implementations are grouped by type and ordered the same way they are declared, to make it easier to navigate the file; this is the section for `NSResponder` overrides (see `#pragma mark`). This should go at the end of the `Private methods` pragma section (corresponding to where you will be adding the declaration).